docs/docs/source/push-changes.html

```diff
@@ -0,0 +1,353 @@
+
```

Comment:
`docs/docs/source/push-changes.html` is it expected?

Reply:
yes, this is expected since the source data for this document is in the "docs/" folder to begin with, the book makes a sort of html copy of the repo, that then resides in "docs/". So "docs/docs/..." is expected. but we can change this a bit if we stop relying on gh-pages to host.

Reply:
eventually, we can also move the site content to the root of a separate gh-pages branch of this repo, so that the repository content and the site content stay segregated and we do not have a nested directory structure like `docs/docs`